<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>571</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_Send_Mes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Emoji</family>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>631</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Send_Mes">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Emoji</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>âœ‰</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>âš™</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Emoji">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ðŸ˜Š</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
